packages/cosmic-swingset/lib/ag-solo/web.js

```diff
@@ -21,9 +21,22 @@ const send = (ws, msg) => {
 
 export function makeHTTPListener(basedir, port, host, rawInboundCommand) {
   // Enrich the inbound command with some metadata.
-  const inboundCommand = (body, { headers: { origin } = {} } = {}) => {
-    const requestContext = { origin, date: Date.now() };
-    return rawInboundCommand({ ...body, requestContext });
+  const inboundCommand = (
+    body,
+    { url, headers: { origin } = {} } = {},
```

Comment:
Why is there a double default value here? What happens if you omit the inner one?
```
{ url, headers: { origin } } = {},
```

Reply:
If `headers` doesn't exist, then the attempt to destructure `origin` will throw an exception.